js/module.3game.js

```diff
@@ -0,0 +1,53 @@
+const getUserChoice = userInput => {
+    userInput = userInput.toLowerCase();
+    if (userInput === 'камень' || userInput === 'ножницы' || userInput === 'бумага' || userInput === 'колодец') {
+        return userInput;
+    } else {
+        console.warn('Ошибка!');
+    }
+}
+
+const getComputerChoice = () => {
+    let computerInput = Math.floor(Math.random() * 3);
+
+    switch (computerInput) {
+        case 0:
+            return 'камень';
+        case 1:
+            return 'ножницы';
+        case 2:
+            return 'бумага';
+    }
+}
+
+
+const determineWinner = (userChoice, computerChoice) => {
+    let comb = userChoice + computerChoice
```

Comment:
Very implicit implementation, don't do that, please. Your code should be easy to read for humans as well as for computers